diff --git a/libc/sysdeps/linux/mips/bits/atomic.h b/libc/sysdeps/linux/mips/bits/atomic.h
index ed8369a..62de520 100644
--- a/libc/sysdeps/linux/mips/bits/atomic.h
+++ b/libc/sysdeps/linux/mips/bits/atomic.h
@@ -37,7 +37,7 @@ typedef uintptr_t uatomicptr_t;
 typedef intmax_t atomic_max_t;
 typedef uintmax_t uatomic_max_t;
 
-#if _MIPS_SIM == _ABIO32 && __mips < 2
+#if 1
 #define MIPS_PUSH_MIPS2 ".set	mips2\n\t"
 #else
 #define MIPS_PUSH_MIPS2
